### ISSUE TITLE
Request - Light Gray Colour

### ISSUE THREAD

**Issue by MoonFlare320** [judemille/GrayEuroScope#1]
Hi there,

great addition to EuroScope!
As you mentioned in the description already: I just wanted to express my interest in having this with a light grey colour (e.g. HTML: #4E5559) to match better with the TopSky-Setup many European Subdivisions (ESAA, LOVV, EGTT, just to name a few) use. 

Thanks!

**Comment by judemille** [comment]
That can certainly be done at some point. A color palette would need to be worked out (primary, secondary, tertiary, blah blah blah), but once that's done, now that the primary legwork has been complete, it should be relatively straight forward. I can't give a timeline, but honestly once a color palette is worked out, it should take no more than a couple days.

**Comment by MoonFlare320** [comment]
That sounds great - what different colors would be required for a complete palette? 
As in: What will the different colors represent?

**Comment by judemille** [comment]
Here's an example of one of the images:
![198](https://github.com/judemille/GrayEuroScope/assets/8127111/3cfb34cb-9059-4535-92da-9b74f4f26d9b)
This encompasses I think the entire five-color palette: really a primary and secondary plus shading.

**Comment by spacenano** [comment]
It would be great to see a more comprehensive guide for this as well so people can make their own colour schemes! I would love a black theme for the TC/NODE displays we use in the UK.

**Comment by judemille** [comment]
I mean, if you're comfortable with SRE and editing over a hundred images in a row, I'd be happy to give you more guidance -- it's just that doing this does require a mild amount of reverse engineering to get to the point where a delta patch can be made.

**Comment by MoonFlare320** [comment]
I have been looking to the "five-color palette" over the last few days, and, for the default one:

I can figure out these two colors: 
#262626 / #6d6d6d

However, I can't find out when these colors are used/how they are defined:
#4b4b4b / #131313

**Comment by judemille** [comment]
Right, so it's a thing of manual shading/dithering -- if you look at that image example I posted above, it should contain each of the colors (I think).

**Comment by judemille** [comment]
I've received a color palette on the VATSIM Scandinavia Discord. I'm working on producing a light gray version now.

**Comment by maxlk96** [comment]
Any progress on this? :)

**Comment by spacenano** [comment]
> Any progress on this? :)

https://github.com/19wintersp/custom-euroscope you can customise the colours here
